pyexchange/api.py

```diff
@@ -10,6 +12,10 @@ class PyexAPI:
     Define a common abstract API for exchanges
     """
 
+    last_nonce = 0
```

Comment:
I believe this needs to be `self.last_nonce`

Reply:
Good point; that's the difference between class and instance variables.  If he made these instance variables, there would be a different nonce for each instance of the class, which might not be what is desired.

As implemented they are class variables, which means the lock would be shared across instances of the class.  If they were implemented in a subclass, that would make sense, because multiple instances for that exchange should share the same lock, not have their own.  But here the class variable is in the baseclass.  If the user has two `PyexAPI` instances which are for different exchanges (read: different subclases), they shouldn't share the same nonce.